azext_iot/tests/central/json/export_webhook.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Regarding the file name, this is no an export to webhook.

Reply:
removed.